### PR TITLE
Add CRUD operations for credit card list

### PR DESCRIPTION
**Pull request by GlendonNotGlen**

### CODE REVIEW

File: src/main/java/seedu/mindmymoney/command/DeleteCommand.java [hunk 1]
```diff
@@ -66,4 +83,43 @@ public void executeCommand() throws MindMyMoneyException {
...
+            if (splitMessage.length != 3) {
+                throw new MindMyMoneyException(System.lineSeparator() + "Please input a number\n"
+                        + "For eg. 'delete /cc 2' to remove the second credit card on your list.\n");
+            }
+            String getNumber = splitMessage[2];
+            int positionToDelete = Integer.parseInt(getNumber) - 1;
+            if (checkCreditCardListBounds(positionToDelete)) {
+                throw new MindMyMoneyException("Please input a valid index");
+            } else {
+                assert positionToDelete >= 0 : "Index should always be >= 0";
```
Comment (SeanHoWB) [inline src/main/java/seedu/mindmymoney/command/DeleteCommand.java:107]: some magic literals here, isit possible to extract them out?

File: src/main/java/seedu/mindmymoney/command/DeleteCommand.java [hunk 2]
```diff
@@ -35,14 +43,23 @@ public boolean checkOutOfBounds(int positionToDelete) {
         return (positionToDelete + 1 <= 0 || positionToDelete + 1 > expenditureList.size());
     }
 
+    /**
+     * Checks if the position to delete is within the bounds of the credit card list.
+     * @param positionToDelete position of index to delete.
+     * @return true if position is within the bounds, false otherwise.
+     */
+    public boolean checkCreditCardListBounds(int positionToDelete) {
+        return (positionToDelete + 1 <= 0 || positionToDelete + 1 > creditCardList.size());
```
Comment (SeanHoWB) [inline src/main/java/seedu/mindmymoney/command/DeleteCommand.java:52]: can change the magic literal 1 and 0?

**Review by SeanHoWB — COMMENTED** [review]
Overall looks good! some magic literals i noticed

File: src/main/java/seedu/mindmymoney/Parser.java [hunk 3]
```diff
@@ -35,25 +36,25 @@ public static Command parseCommand(String input, ExpenditureList itemList, Credi
 
             switch (parsedInput[INDEX_OF_FIRST_ITEM_IN_STRING].toLowerCase()) {
             case "help":
-                return new HelpCommand(true);
+                return new HelpCommand(true, parsedInput[INDEX_OF_SECOND_ITEM_IN_STRING]);
```
Comment (khseah) [inline src/main/java/seedu/mindmymoney/Parser.java:39]: As mentioned, maybe we stick to 1 help command that lists everything

File: src/main/java/seedu/mindmymoney/command/ListCommand.java [hunk 4]
```diff
@@ -41,15 +58,52 @@ public boolean isExit() {
...
+        }
+        assert listInString.length() != 0 : "Return string should be non-empty";
+        return listInString;
+    }
+
+    /**
+     * Prints user's current list of credit cards.
+     */
+    public void printCreditCardList() throws MindMyMoneyException {
+        if (creditCardList.size() == 0) {
```
Comment (khseah) [inline src/main/java/seedu/mindmymoney/command/ListCommand.java:88]: Maybe can change to use the .isEmpty() method instead, then can avoid the magic literal 0 also
```suggestion
        if (creditCardList.isEmpty()) {
```

**Review by khseah — COMMENTED** [review]
Nice job!

**Pull request merged**